src/python/foglamp/configuration_manager.py

```diff
@@ -29,7 +31,8 @@
 
 _valid_type_strings = ['boolean', 'integer', 'string', 'IPv4', 'IPv6', 'X509 certificate', 'password']
```

Comment:
password is fine to keep in _valid_type_strings. 
Ivan's concern was regarding this line:
_connection_string = 'postgresql://foglamp:foglamp@localhost:5432/foglamp'

We need to change this and all other connection strings to connect to the DB via socket connection. See:
https://docs.google.com/document/d/1QDkwb1l3XPZCTZfBVwmchgI9ay0vmfKNXUEwu1KcuOA/edit#heading=h.omckl7aauz2g